ServerCore/Pages/Teams/Details.cshtml

```diff
@@ -204,26 +204,26 @@
             @if (Model.Event.EventHasTeamSwag)
             {
                 <div class="row">
-                    <h2>Pizza</h2>
+                    <h2>Lunch</h2>
                     <p>
-                        Your team can currently have @Model.EditableLunches pizza(s). If your team wants to opt out of a pizza, select &quot;No Pizza&quot;. If you cannot have any of the options offered due to dietary restrictions, please email <a href="mailto:puzzhunt@microsoft.com">puzzhunt@microsoft.com</a> with details and we'll find an alternative.
+                        Your team can currently have @Model.EditableLunches lunch(s). If your team wants to opt out of one of the lunches, select &quot;No Lunch&quot;. If you cannot have any of the options offered due to dietary restrictions, please email <a href="mailto:puzzhunt@microsoft.com">puzzhunt@microsoft.com</a> with details and we'll find an alternative.
```

Comment:
The use of "lunch" in this section sounds more individual than shared, and I suggest this page use the word "item" to be more generic and avoid potential confusion.